### PR TITLE
update dev with new template

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: lib/NfcoreTemplate.groovy [hunk 1]
```diff
@@ -24,7 +24,7 @@ class NfcoreTemplate {
     public static void checkConfigProvided(workflow, log) {
         if (workflow.profile == 'standard' && workflow.configFiles.size() <= 1) {
             log.warn "[$workflow.manifest.name] You are attempting to run the pipeline without any custom configuration!\n\n" +
-                    "This will be dependent on your local compute environment but can be achieved via one or more of the following:\n" +
+                    "This will be dependent on your local compute enviroment but can be acheived via one or more of the following:\n" +
```
Comment (erikrikarddaniel) [inline lib/NfcoreTemplate.groovy]: Funny if this is the updated template -- new spelling mistakes!?

File: nextflow.config [hunk 2]
```diff
@@ -134,6 +134,13 @@ profiles {
     test_transdecoder   { includeConfig 'conf/test_transdecoder.config' }
 }
 
+// Load igenomes.config if required
+if (!params.igenomes_ignore) {
+    includeConfig 'conf/igenomes.config'
+} else {
+    params.genomes = [:]
+}
+
```
Comment (erikrikarddaniel) [inline nextflow.config]: I can't see that we want this, we're not using igenomes.

File: workflows/metatdenovo.nf [hunk 3]
```diff
@@ -49,6 +49,7 @@ include { UNPIGZ as UNPIGZ_MEGAHIT_CONTIGS } from '../modules/local/unpigz.nf'
 // SUBWORKFLOW: Consisting of a mix of local and nf-core/modules
 //
 include { INPUT_CHECK } from '../subworkflows/local/input_check'
+<<<<<<< HEAD
```
Comment (erikrikarddaniel) [inline workflows/metatdenovo.nf]: Merge conflict line.

**Review by erikrikarddaniel — CHANGES_REQUESTED** [review]
A few mistakes. Plus you seem to have a merge conflict in `conf/modules.config`.

**Comment by erikrikarddaniel** [comment]
There's still a merge conflict in `modules.config` it seems.

**Review by emnilsson — APPROVED** [review]
LGTM

**Pull request merged**